Jenkinsfile.conda

```diff
@@ -1,2 +1,44 @@
-@Library('JenkinsShared')_
-CondaPipeline("[ts_config_ocs]", "ts-genericcamera", "lsst.ts.GenericCamera")
+pipeline {
```

Comment:
We need to talk to Rob and Eric about this.

Reply:
I agree.